modules/core/src/smithy4s/syntax/package.scala

```diff
@@ -113,4 +113,8 @@ package object syntax
       OneOf(oneOf.label, oneOf.schema.withHints(hints: _*), oneOf.inject)
   }
 
+  implicit class withMaskSyntax[F[_]](val s: Schematic[F]) extends AnyVal {
```

Comment:
s should be private, otherwise you can do schematic.s.s.s.s.s I think

Reply:
Mmm not sure it should leave there tbh, nor that it should be a extension, but 🤷

Reply:
Fair enough, I think it has proved less useful than I thought it would initially anyway